opencti-platform/opencti-front/src/private/components/common/files/workbench/WorkbenchFileViewer.jsx

```diff
@@ -36,6 +42,8 @@ const WorkbenchFileViewerBase = ({
   const { id, pendingFiles } = entity;
   const { edges } = pendingFiles;
   const isContainer = entity.entity_type !== 'Report';
```

Comment:
I think this is a miss from a LONG time ago. We have many more container than just reports, can you check that ?

Reply:
isContainer is only used to set `disableImport` of WorkbenchFileLine, but `disableImport` doesn't seem to be used at all in WorkbenchFileLine... do you know why ? @Kedae I guess we should just remove it ?

Reply:
Yep, didn't saw that. You can simply remove it

Reply:
Done